Content.Server/Strip/StrippableSystem.cs

```diff
@@ -264,6 +271,7 @@ bool Check()
 
             _handsSystem.TryDrop(user, checkActionBlocker: false, handsComp: userHands);
             _handsSystem.TryPickup(component.Owner, held, handName, checkActionBlocker: false, animateUser: true, handsComp: hands);
+            _adminLogger.Add(LogType.Stripping, LogImpact.Medium, $"{ToPrettyString(user):player} has placed the item {ToPrettyString(held):item} in {ToPrettyString(component.Owner):player}'s hands");
             // hand update will trigger strippable update
```

Comment:
I'm pretty sure having to ":player" labels causes issues somewhere. They should all be distinct (e.g., user & target).

Reply:
Oooh, they're labels? I thought they were descriptors, as if I was telling the function what type of info it was looking for. Okie dokie, I shall fix.

Reply:
TBH I can't fully remember what they're for, but I know they have some purpose. I asked in the maintainer chat on discord so maybe check there in case someone responds with more info.